tasks/rsyslog.yml

```diff
@@ -0,0 +1,19 @@
+- name: Install Rsyslog custom PPA for a consistent version
+  apt_repository:
+    repo: "{{ loggly.rsyslog_ppa }}"
```

Comment:
What about making this var name (`loggly.rsyslog_ppa`) a little more generic since it could be any additional apt-repository (and not just a PPA)? I think we could then also say this role is distribution agnostic.